### PR TITLE
Removing duplicate enumeration (thanks to David)

### PR DESCRIPTION
**Pull request by skinkie**
Fix #372  (Directly on master)

### CODE REVIEW

**Comment by skinkie** [comment]
@Aurige There is a change in an enum being pantographABove to pantograph, I don't see pantograph ever used in the documentation.

**Review by ue71603 — APPROVED** [review]

**Review by trurlurl — APPROVED** [review]
I cannot judge, however, whether the depreciation note wasn't perhaps justified...

**Review by Aurige — COMMENTED** [review]
I would prefer keeping pantographABove (once only of course)

**Comment by skinkie** [comment]
> I would prefer keeping pantographABove (once only of course)

That would certainly be a typo. pantographAbove is something I can live with.

File: xsd/netex_framework/netex_reusableComponents/netex_nm_equipmentEnergy_support.xsd [hunk 1]
```diff
@@ -214,8 +214,11 @@ Rail transport, Roads and Road transport
 		<xsd:restriction base="xsd:NMTOKEN">
 			<xsd:enumeration value="undefined"/>
 			<xsd:enumeration value="plug"/>
-			<xsd:enumeration value="pantographABove"/>
-			<xsd:enumeration value="pantographABove"/>
+			<xsd:enumeration value="pantograph">
+				<xsd:annotation>
+					<xsd:documentation>was pantographABove before.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
```
Comment (ue71603) [inline xsd/netex_framework/netex_reusableComponents/netex_nm_equipmentEnergy_support.xsd:221]:
```suggestion
			<xsd:enumeration value="pantographAbove">
				<xsd:annotation>
					<xsd:documentation>depreciated. Will be panthograph.</xsd:documentation>
				</xsd:annotation>
			</xsd:enumeration>
			<xsd:enumeration value="pantograph">
			</xsd:enumeration>
```

**Review by Aurige — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Pull request merged**